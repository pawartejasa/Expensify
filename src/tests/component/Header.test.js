import { shallow } from 'enzyme';
import React from 'react';
import Header from '../../Components/Header';
import toJSON from 'enzyme-to-json';
test('Should render header correctly', ()=>{
    const wrapper= shallow(<Header />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});