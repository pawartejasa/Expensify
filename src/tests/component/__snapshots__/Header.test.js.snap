// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header correctly 1`] = `
<header>
  <h1>
    Exemplify
  </h1>
  <NavLink
    activeClassName="active"
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/edit"
  >
    Edit Expenses
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help Expenses
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create Expenses
  </NavLink>
</header>
`;
